### PR TITLE
Make minor docs changes for 1.2 release

### PR DESCRIPTION
**Pull request by echeran**

### CODE REVIEW

**Comment by Manishearth** [comment]
You need to run `cargo make generate-readmes`

**Review by Manishearth — APPROVED** [review]

File: components/timezone/README.md [hunk 1]
```diff
@@ -49,8 +49,7 @@ As with time zone IDs, there are two interchangeable forms:
 ICU4X uses the short form.
 
 Note: in ICU4X, "metazone" is one word and "time zone" is two words, except for this crate
-and module name, where "timezone" is used with no separators. See
-<https://github.com/unicode-org/icu4x/issues/2507>.
```
Comment (sffc) [inline components/timezone/README.md:53]: Question: Why did you remove the link? Removing cross-references seems like a regression to me.

File: components/properties/src/provider/bidi_data.rs [hunk 2]
```diff
@@ -120,7 +120,7 @@ impl TryFrom<u32> for MirroredPairedBracketData {
 }
 
 /// A closed Rust enum representing a closed set of the incoming Bidi_Paired_Bracket_Type
-/// property values necessary in the internal representation of [`MirroredPairedBracketData`]
+/// property values necessary in the internal representation of `MirroredPairedBracketData`
```
Comment (sffc) [inline components/properties/src/provider/bidi_data.rs:123]: Question: why did you remove the link?

**Review by sffc — COMMENTED** [review]
Post-submit questions
